substra/sdk/client.py

```diff
@@ -164,9 +163,20 @@ def add_data_sample(self, data, local=True, timeout=False,
             "test_only": bool,
         }
 ```
+        The `path` in the data dictionary must point to a directory representing the
+        data sample content. Note that the directory can contain multiple files, all the
+        directory content will be added to the platform.
+
+        If `local` is true, `path` must refer to a directory located on the local
+        filesystem. The file content will be transferred to the server through an
+        HTTP query, so this mode should be used for relatively small files (<10mo).
+
+        If `local` is false, `path` must refer to a directory located on the server
+        filesystem. This mode is well suited for all kind of file sizes.
```

Comment:
Same as previous comment, the `path` should be accessible (readable) for substra-backend.

Reply:
Done